src/windows/component.rs

```diff
@@ -11,15 +13,12 @@ use windows::Win32::System::Com::{
     RPC_C_IMP_LEVEL_IMPERSONATE,
 };
 use windows::Win32::System::Rpc::{RPC_C_AUTHN_WINNT, RPC_C_AUTHZ_NONE};
-use windows::Win32::System::Variant::VariantClear;
+use windows::Win32::System::Variant::{VariantClear, VARIANT};
 use windows::Win32::System::Wmi::{
     IEnumWbemClassObject, IWbemLocator, IWbemServices, WbemLocator, WBEM_FLAG_FORWARD_ONLY,
     WBEM_FLAG_NONSYSTEM_ONLY, WBEM_FLAG_RETURN_IMMEDIATELY, WBEM_INFINITE,
 };
 
-use std::cell::OnceCell;
```

Comment:
These two types are still used by `SECURITY`. I really wonder how it's possible for the CI to succeed.

Reply:
Ah? Weird. Maybe a new version of rustfmt...

Reply:
Or maybe I was lying... I guess I sometimes order things alphabetically without thinking...